online-resources/free-ebooks.md

```diff
@@ -0,0 +1,8 @@
+# Great free ebooks to learn to program and grasp computer science concepts
```

Comment:
Nice idea. I'll merge this now, but could it even be a separate file if we gather enough? A bit like the podcasts file?